init_replication.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+docker exec -i master psql -U postgres -d main -c "CREATE PUBLICATION my_publication FOR ALL TABLES;"
```

Comment:
Move it to bin. And it should execute automatically on `docker-compose up` spage!